examples/mugen/data/audio_utils.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import av
+import numpy as np
+
+
+AUDIO_SAMPLE_RATE = 22050
+AUDIO_SAMPLE_LENGTH = AUDIO_SAMPLE_RATE * 3.2  # each video is 3.2 seconds
+
+
```

Comment:
Is this from Jukebox? Can we add a url to the reference code for tracking purpose?

Reply:
it's from MUGEN in their [audio vqvae](https://github.com/mugen-org/MUGEN_baseline/blob/main/lib/models/audio_vqvae/hparams.py#L7)

Reply:
the load_audio function is from jukebox so i'll add that url too